src/main/python/rlbot/agents/base_script.py

```diff
@@ -27,6 +33,13 @@ def __init__(self, name):
         self.game_interface.load_interface()
         fake_index = random.randint(100, 10000)  # a number unlikely to conflict with bots or other scripts
         self.renderer = self.game_interface.renderer.get_rendering_manager(bot_index=fake_index, bot_team=2)
+        # Get matchcomms root if provided as a command line argument.
+        try:
+            pos = sys.argv.index("--matchcomms-url")
+            potential_url = sys.argv[pos + 1]
+            self.matchcomms_root = urlparse(potential_url)  # Not sure what to do if this is not a valid url
+        except (ValueError, IndexError):
+            pass
```

Comment:
In my opinion, it would be good to fail loud and fast here, by just letting the exception bubble up.

Reply:
Those error are related to the args checking. `urlparse` doesn't raise any errors.